db/migrate/20160126151325_add_disk_and_memory_minimum_to_hardware.rb

```diff
@@ -0,0 +1,6 @@
+class AddDiskAndMemoryMinimumToHardware < ActiveRecord::Migration
+  def change
+    add_column :hardwares, :disk_size_minimum, :integer, :limit => 8
```

Comment:
@Ladas I agree, this is always confusing.  I guess either the column name should include GB or the refresh should save as bytes.